src/request.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const http = require('http');
+
+const BASE = 'http://localhost:5700';
+
+const href = BASE + '/hello-world?toCase=CAMEL';
```

Comment:
The 'href' variable is hardcoded. This doesn't comply with the task requirement to accept requests in the format '/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>'. The parameters should not be hardcoded and should be designed to accept dynamic input.